tests/diagnostics/snapshots/test_advanced_formatting.txt

```diff
@@ -0,0 +1,7 @@
+Error: Can't compare apples with oranges (at <unknown>:1:0)
+  | 
+1 | apple == orange
+  | ^^^^^ This is an apple
+  | 
+1 | apple == orange
+  |          ------ This is not an apple
```

Comment:
Shouldn't this have printed "aple", since the subdiagnostic overwrites the `pp` attribute ?

Reply:
Ah, nvm. The format string is "{p}{pp}".